Magento_Customer/templates/widget/name.phtml

```diff
@@ -78,7 +78,7 @@ $suffix = $block->showSuffix();
                         value="<?= $_option ?>"
```

Comment:
escapeHtmlAttr

Reply:
plus compare this file with original file. I see some missing escapeHtmlAttr for some attirbutes here like id, name, title etc.